api/configs/middleware/__init__.py

```diff
@@ -176,14 +176,28 @@ def SQLALCHEMY_DATABASE_URI(self) -> str:
         default=os.cpu_count() or 1,
     )
 
-    @computed_field
+    @computed_field  # type: ignore[misc]
+    @property
     def SQLALCHEMY_ENGINE_OPTIONS(self) -> dict[str, Any]:
+        # Parse DB_EXTRAS for 'options'
+        db_extras_dict = dict(parse_qsl(self.DB_EXTRAS))
+        options = db_extras_dict.get("options", "")
+        # Always include timezone
+        timezone_opt = "-c timezone=UTC"
+        if options:
+            # Merge user options and timezone
+            merged_options = f"{options} {timezone_opt}"
```

Comment:
I’m wondering how we should handle merging the options in cases where the user explicitly sets `timezone`. Should we prioritize the user’s configuration in this scenario? 

It seems that ensuring no duplicate options would require parsing the `options` as command-line arguments, which could introduce compatibility challenges.

Reply:
We use UTC time extensively in our code to achieve internationalization. I think UTC should be the default option for our database configuration.